engine/sim/sim.cpp

```diff
@@ -2137,6 +2141,7 @@ void sim_t::analyze_error()
 
   double mean_total=0;
   int mean_count=0;
+  double current_standard_error = 0.0;
```

Comment:
value of this variable outside of single_actor_batch sims is never set, leading to t_test always returning a se of 0. Thus, breaking the t_test culling logic.

Reply:
Thanks - I'll have some time on the weekend to work on this. I'll make a change and push a patch once I have it.